### PR TITLE
Add predefined queries for testing

### PR DESCRIPTION
**Pull request by StanHus**
Relation: 
https://eaflood.atlassian.net/jira/software/projects/FFAFGB/boards/1631?selectedIssue=FFAFGB-121

### CODE REVIEW

File: app/routes/conversation.js [hunk 1]
```diff
@@ -132,5 +133,78 @@ module.exports = [
...
+
+        logger.info('Response generated in ' + ((endTime.getTime() - startTime.getTime()) / 1000).toFixed(2) + ' seconds')
+      }
+
+      // return response as an html table
+      const table = responses.map((response) => {
+        return `<tr><td>${response.input}</td><td>${response.response}</td><td>${response.responseDuration}</td></tr>`
+      }).join('')
+
+      const html = `
```
Comment (MohammedFoysal) [inline app/routes/conversation.js:176]: Can you put this in an njk file

Comment (StanHus) [reply app/routes/conversation.js]: done

**Comment by StanHus** [comment]
Testing by going to http://localhost:3000/test_prompts

**Review by MohammedFoysal — APPROVED** [review]

File: app/routes/conversation.js [hunk 2]
```diff
@@ -137,5 +138,70 @@ module.exports = [
         schemesList
       })
     }
+  },
+  {
+    method: 'GET',
+    path: '/test_prompts',
+    handler: async (request, h) => {
+      if (process.env.ENDPOINT_TESTING_ENABLED !== 'true') {
```
Comment (MohammedFoysal) [inline app/routes/conversation.js:146]: Please put it in the config file app/config/index.js

File: appConfig/appConfig.snd3.yaml [hunk 3]
```diff
@@ -60,3 +60,6 @@
   type: "keyvault"
 - key: GOOGLE_TAG_MANAGER_KEY
   value: "G-NF25VDMQQX"
+- key: ENDPOINT_TESTING_ENABLED
```
Comment (MohammedFoysal) [inline appConfig/appConfig.snd3.yaml:63]: don't forget to add to build.yaml

**Comment by StanHus** [comment]
Updated snd3 and snd4, pinged Rama to test

File: app/routes/conversation.js [hunk 4]
```diff
@@ -137,5 +138,70 @@ module.exports = [
         schemesList
       })
     }
+  },
+  {
+    method: 'GET',
+    path: '/test_prompts',
```
Comment (MohammedFoysal) [inline app/routes/conversation.js:144]: This should be in a new routes file

**Review by MohammedFoysal — APPROVED** [review]

**Pull request merged**